### PR TITLE
Added Longitude, Latitude, Altitude on Define Climatic Dialog

### PR DESCRIPTION
**Pull request by MeSophie**
Fixes Partially #8137

@rdstern I done the first step of this issue.

@lloyddewit, @N-thony  Please can you have a look ?

### CODE REVIEW

File: instat/DlgDefineClimaticData.vb [hunk 1]
```diff
@@ -48,6 +48,9 @@ Public Class DlgDefineClimaticData
         Dim kvpRain As KeyValuePair(Of String, List(Of String)) = New KeyValuePair(Of String, List(Of String))("rain", {"rain", "prec", "rr", "prcp"}.ToList())
         Dim kvpDate As KeyValuePair(Of String, List(Of String)) = New KeyValuePair(Of String, List(Of String))("date", {"date", "record"}.ToList())
         Dim kvpStation As KeyValuePair(Of String, List(Of String)) = New KeyValuePair(Of String, List(Of String))("station", {"station", "id", "name"}.ToList())
+        Dim kvpAltitude As KeyValuePair(Of String, List(Of String)) = New KeyValuePair(Of String, List(Of String))("altitude", {"altitude", "elevation", "elev", "alt"}.ToList())
+        Dim kvpLatitude As KeyValuePair(Of String, List(Of String)) = New KeyValuePair(Of String, List(Of String))("longitude", {"longitude", "lon"}.ToList())
+        Dim kvpLongitude As KeyValuePair(Of String, List(Of String)) = New KeyValuePair(Of String, List(Of String))("latitude", {"latitude", "lat"}.ToList())
```
Comment (lloyddewit) [inline instat/DlgDefineClimaticData.vb:53]: swap names?
```suggestion
        Dim kvpLongitude As KeyValuePair(Of String, List(Of String)) = New KeyValuePair(Of String, List(Of String))("longitude", {"longitude", "lon"}.ToList())
        Dim kvpLatitude As KeyValuePair(Of String, List(Of String)) = New KeyValuePair(Of String, List(Of String))("latitude", {"latitude", "lat"}.ToList())

```

**Review by lloyddewit — CHANGES_REQUESTED** [review]

**Review by rdstern — COMMENTED** [review]
@MeSophie this is looking very promising.  Can you try with the file from the library with the 4 stations linked data.  This will need some correction, but that is not an issue for this dialogue.
a) The linking variable is the station name, so that should be detected.  I assume you use the data file first.  Then go to the

**Comment by MeSophie** [comment]
> @MeSophie this is looking very promising. Can you try with the file from the library with the 4 stations linked data. This will need some correction, but that is not an issue for this dialogue. a) The linking variable is the station name, so that should be detected. I assume you use the data file first. Then go to the

@rdstern Sorry it look like you didn't finished your instructions.

**Comment by N-thony** [comment]
> @MeSophie this is looking very promising. Can you try with the file from the library with the 4 stations linked data. This will need some correction, but that is not an issue for this dialogue. a) The linking variable is the station name, so that should be detected. I assume you use the data file first. Then go to the

@rdstern could you complete your reviews/comments?

**Comment by MeSophie** [comment]
@N-thony  I resolved the bug with check duplicate. This PR is ready for review.

**Comment by N-thony** [comment]
> @N-thony I resolved the bug with check duplicate. This PR is ready for review.

@MeSophie looks good, @rdstern can you test?

**Pull request closed without merging**